website/src/components/content/WhitePaperPage.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import { RequestDemoAction } from '../../css/components/actions/RequestDemoAction'
+import { COLORS } from '../Jumbotron'
+import { ContentPage } from './ContentPage'
+
+interface Quote {
+    quote: string
+    author: string
+    image?: string
+}
+
+interface Props {
+    title: string
+    quote?: Quote
+    pdf?: string
+    className?: string
+    titleClassName?: string
+    children?: React.ReactNode
+    bgImage?: string
+}
+
+export const WhitePaperJumbotron: React.FunctionComponent<{
+    className?: string
+    color?: keyof typeof COLORS
+    children?: React.ReactNode
+}> = ({ className = '', color = 'dark', children }) => (
+    <div className={`jumbotron rounded-0 ${COLORS[color]} ${className}`}>
+        <div className="container text-center pt-5 pb-3">
+            <span className="white-papers__label d-block mt-1">white paper</span>
+            {children}
+        </div>
+    </div>
+)
+
+export const MediaQuote: React.FunctionComponent<{
+    image?: string
+    quote: string
+    author: string
+}> = ({ image, quote, author }) => (
+    <div className="container pt-6">
+        <div className="case-studies__quote row justify-content-center">
+            {image && (
+                <div className="col-12 col-lg-9">
+                    <img className="rounded-circle img-fluid mx-auto d-block pb-3" src={image} alt={author} />
+                </div>
+            )}
+            <div className="col-12 col-lg-9">
+                <blockquote className="blockquote">
+                    <p className="text-light" dangerouslySetInnerHTML={{ __html: quote }} />
+                    <footer className="blockquote-footer text-light mt-1">{author}</footer>
+                </blockquote>
+            </div>
+        </div>
+    </div>
+)
+
+export const WhitePaperPage: React.FunctionComponent<Props> = ({
+    title,
+    quote,
+    className = 'white-paper',
+    titleClassName = '',
+    pdf,
+    children,
+    bgImage,
+}) => (
+    <div
+        className={`${className}`}
+        // tslint:disable-next-line: jsx-ban-props
+        style={bgImage ? { backgroundImage: `url(${bgImage})`, backgroundRepeat: `no-repeat` } : undefined}
+    >
+        <WhitePaperJumbotron className="mb-5">
+            <h1 className={`${titleClassName}`}>{title}</h1>
+            {pdf && (
+                <a href={pdf} className="btn btn-primary mt-4" rel="nofollow" target="_blank">
+                    <i className="fa fa-file-pdf pr-2" />
+                    Download PDF
+                </a>
+            )}
+        </WhitePaperJumbotron>
+        {children}
+    </div>
+)
+
+export const InContentBlockquote: React.FunctionComponent<{
```

Comment:
There are a few similarities between the new whitepaper components and existing case study components. If possible, it would be good to create a single higher-level component that could be reused across both.

Reply:
Yes, that would be the better approach.

Reply:
Created https://github.com/sourcegraph/about/issues/750